deployment/helm/resource-management-policies/topology-aware/values.yaml

```diff
@@ -0,0 +1,25 @@
+# Default values for nri-plugins.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+---
+image:
+  name: ghcr.io/containers/nri-plugins/nri-resource-policy-topology-aware
+  # tag, if defined will use the given image tag, otherwise Chart.AppVersion will be used
+  tag: unstable
+  pullPolicy: Always
+
+namespace: kube-system
+
+config:
+  Active: topology-aware
+  ReservedResources:
+    cpu: 750m
+
+metrics:
+  port: 8891
+  protocol: TCP
+  hostPort: 8891
+
+quota:
```

Comment:
Was there discussion about this, why is it called quota? I myself was immediately thinking about ResourceQuota

Reply:
no specific reason. I will rename it